src/util/rocksdb_util/properties.rs

```diff
@@ -364,11 +377,8 @@ impl SizePropertiesCollector {
 
 impl TablePropertiesCollector for SizePropertiesCollector {
     fn add(&mut self, key: &[u8], value: &[u8], entry_type: DBEntryType, _: u64, _: u64) {
-        if entry_type != DBEntryType::Put {
-            return;
-        }
-        let size = key.len() + value.len();
-        self.index_handle.size += size as u64;
+        let size = key.len() as u64 + get_entry_size(value, entry_type).unwrap();
```

Comment:
When `entry_type` is `DBEntryType::Delete`, `get_entry_size` will return Err(()) which will cause panic here.

Reply:
In that case, maybe it's more straight forward to put the code back here :(